targets/drive/mobile/fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,11 +1,14 @@
-This application is only compatible with the latest version of Cozy (v3).
+We all have a life and a digital life. Cozy gathers them. In your Cozy, you have all your personal data: pictures, health, bank, holidays, accessible only by you.
+Here comes the Cozy first mobile app, in its Beta version (replacing the old V2 version): Cozy Drive.
```

Comment:
Are we still convinced that `Beta version` is a good thing to do. So 2010. Nowadays we create app v1 that is continuously enhanced.

Reply:
Yes we are ;)

Reply:
`Here comes the Cozy first mobile app` -> `Here comes the first Cozy mobile app` ?

Reply:
Yup